tests/snippetgen/goldens/mollusca_generated_mollusca_v1_snippets_method_bidi_streaming_async.py

```diff
@@ -34,11 +34,12 @@ async def sample_method_bidi_streaming():
     client = mollusca_v1.SnippetsAsyncClient()
 
     # Initialize request argument(s)
-    request = mollusca_v1.SignatureRequest(
+    request = mollusca_v1.SignatureRequestOneRequiredField(
+        a_string="a_string_value",
     )
 
     # Make the request
-    stream = await client.method_bidi_streaming([])
+    stream = await client.method_bidi_streaming([a_string="a_string_value"])
```

Comment:
nit - is `a_string` "a string" or "a-string" (with the implication of a "b-string" or a "c-string"?) If "a string" and it's something we have the power to change, then I feel like `my_string` or `your_string` is more useful but if not, plz disregard

Reply:
Ah this is from the field naming in the mock proto file.  It is completely changeable, I will do that in a follow up commit.  :D https://github.com/googleapis/gapic-generator-python/blob/cc3de569ac610b3ca60259a53c572e60a6cea180/tests/snippetgen/snippets.proto

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/941/commits/95282b502a08d0c72fb45537cd2ed3fb1853462f